spec/requests/user_status_controller_spec.rb

```diff
@@ -145,9 +144,8 @@
         messages = MessageBus.track_publish { delete "/user-status.json" }
 
         expect(messages.size).to eq(1)
-        expect(messages[0].channel).to eq("/user-status/#{user.id}")
-        expect(messages[0].data).to eq(nil)
-        expect(messages[0].user_ids).to eq([user.id])
```

Comment:
I think it is still useful to assert that the message is published to the right group.